dali/operators/audio/mfcc/mfcc.cu

```diff
@@ -69,7 +69,7 @@ bool MFCC<GPUBackend>::SetupImpl(std::vector<OutputDesc> &output_desc,
     output_desc = detail::SetupKernel<T>(kmgr_, ctx_, input, make_cspan(args_), axis_);
   ), DALI_FAIL(make_string("Unsupported data type: ", input.type().id())));  // NOLINT
   int64_t max_ndct = 0;
-  for (int i = 0; i < nsamples_; ++i) {
+  for (int i = 0; i < input.ntensor(); ++i) {
```

Comment:
In such instances we should always take the _output_ size, not the input.
Rationale: if we use output size, all we risk is reading garbage or illegal read; if we use input size, we run the risk of corrupting memory and the function becomes a potential attack route.